frontend/src/queries/nodes/DataVisualization/Components/Table.tsx

```diff
@@ -17,18 +18,56 @@ interface TableProps {
 }
 
 export const Table = (props: TableProps): JSX.Element => {
-    const { tabularData, tabularColumns, responseLoading, responseError, queryCancelled, response } =
-        useValues(dataVisualizationLogic)
+    const {
+        tabularData,
+        tabularColumns,
+        conditionalFormattingRules,
+        responseLoading,
+        responseError,
+        queryCancelled,
+        response,
+    } = useValues(dataVisualizationLogic)
 
-    const tableColumns: LemonTableColumn<any[], any>[] = tabularColumns.map(({ column, settings }, index) => ({
-        title: settings?.display?.label || column.name,
-        render: (_, data, recordIndex: number) => {
-            return renderColumn(column.name, data[index], data, recordIndex, {
-                kind: NodeKind.DataTableNode,
-                source: props.query.source,
-            })
-        },
-    }))
+    const tableColumns: LemonTableColumn<TableDataCell<any>[], any>[] = tabularColumns.map(
+        ({ column, settings }, index) => ({
+            title: settings?.display?.label || column.name,
+            render: (_, data, recordIndex: number) => {
+                return renderColumn(column.name, data[index].formattedValue, data, recordIndex, {
+                    kind: NodeKind.DataTableNode,
+                    source: props.query.source,
+                })
+            },
+            style: (_, data) => {
```

Comment:
Is this conditional formatting only meant for tables?

Reply:
Yep, only for `Table` display